portal/portal/academy/views.py

```diff
@@ -140,7 +140,8 @@ class StudentUnitDetailView(StudentViewsMixin, BaseUnitDetailView):
 
 class InstructorUserListView(InstructorViewsMixin, ListView):
     model = get_user_model()
-    queryset = get_user_model().objects.filter(is_student=True, failed_or_dropped=False)
+    queryset = model.objects.filter(is_student=True, failed_or_dropped=False)
+    #queryset = get_user_model().objects.filter(is_student=True, failed_or_dropped=False)
```

Comment:
It is ok to make this change, but please don't leave commented code 🙏 You can remove the second line.